private/buf/cmd/buf/command/push/push.go

```diff
@@ -193,7 +193,7 @@ func run(
 	if err != nil {
 		if connect.CodeOf(err) == connect.CodeAlreadyExists {
 			if _, err := container.Stderr().Write(
-				[]byte("The latest commit has the same content; not creating a new commit.\n"),
+				[]byte(fmt.Sprintf("%s\n", err.Error())),
```

Comment:
Why are we special-casing for connect.CodeAlreadyExists at all now?

Reply:
Because this error can come from 2 different places: identical content or existing tag, but it's assuming it's always coming from "identical content". See the PR description.

Reply:
I'm saying how is it any different than `return err`

Reply:
Ah ok, I think it's just making sure to end with a new line.